src/weppy.ts

```diff
@@ -0,0 +1,478 @@
+/// <reference path="../weppy.d.ts"/>
+
+// these seem not to be defined in window
+interface Window {
+	XMLHttpRequest?: XMLHttpRequest;
+	XDomainRequest?: XMLHttpRequest;
+	JSON?: JSON;
+}
+
+module WeppyImpl {
+
+	var protocolVersion = 3,
+		pathDelimiter = '.',
+		namespaceDelimiter = '::',
+		active = false,
+		options: WeppySettings = {
+			"host": '/weppy',
```

Comment:
I specifically put them in quotes to prevent name mangling when using google closure compiler as they need to stay as they are. However it seems we won't use it for minification. Do you think they should stay quoted or not in that case?